int64 clk_val		# Clock value